src/useIsStickyStuck/useIsStickyStuck.ts

```diff
@@ -0,0 +1,36 @@
+import { Ref, computed, onMounted, ref } from 'vue'
+import { MaybeRef } from '@/types/maybe'
+import { useIntersectionObserver } from '@/useIntersectionObserver'
+
+type UseIsStickyStuckOptions = {
+  rootMargin?: string,
+  boundingElement?: HTMLElement | Ref<HTMLElement | undefined>,
+}
+
+export function useIsStickyStuck(
+  element: HTMLElement | Ref<HTMLElement | undefined>,
+  options?: MaybeRef<UseIsStickyStuckOptions>,
+): Ref<boolean> {
```

Comment:
Generally we try and have compositions return objects so in the future we can enhance things without making breaking changes. Can you update this so the syntax is `const { suck } = useIsStickyStuck()`?